.tours/typescript/http/.tours/01creating-a-function-app.tour

```diff
@@ -3,7 +3,7 @@
   "title": "TypeScript HTTP - 01. Creating a Function App",
   "steps": [
     {
-      "file": "../../../src/typescript/AzureFunctions.Http/01CreatingAFunctionApp/HelloWorldHttpTrigger/index.ts",
```

Comment:
Could you revert this change? The tour files are supposed to be used from within a Workspace, that's why the paths is so long. Unfortunately this does mean CodeTours can't be used in the folder itself :/.

Reply:
I reverted the change, but this means the Typescript tour now works only for some steps. All steps that don't use 
"../../../src" wont work when I open the Typescript workspace. For example, I can't run tours 2-4 and steps 10 to the end in Tour 1. Might be a Mac thing. Shall I prefix all paths in the Typescript tour with "../../../src"?

Reply:
Ah ok, yes that's probably an OS specific thing. I'll fix that in a separate PR :).